src/locales/en/translations.json

```
@@ -85,14 +85,14 @@
     "info-link": "<0>Use our contact-form or write an e-mail to <1>beep@satellytes.com</1></0>",
     "info": "Use our contact-form or write an e-mail to beep@satellytes.com",
     "name": "Name",
-    "email": "E-mail-adresse",
+    "email": "E-mail-adress",
     "message": "Your message to us",
     "mandatory-field": "mandatory field",
     "error": {
       "name": "Your name is missing",
       "message": "Your message is missing",
       "email": "Your e-mail is missing",
-      "email-unknown": "Something is wrong with the email-adresse"
+      "email-unknown": "Something is wrong with the e-mail-adress"
```

Same here. It's `address`... and I would drop that second hyphen. However, I very much like the `e-mail` (as opposed to `email`).
```suggestion
      "email-unknown": "Something is wrong with the e-mail address"
```